tests/suites/relations/relation_data_exchange.sh

```diff
@@ -0,0 +1,101 @@
+run_relation_data_exchange() {
+    echo
+
+    model_name="test-relation-data-exchange"
+    file="${TEST_DIR}/${model_name}.txt"
+
+    ensure "${model_name}" "${file}"
+
+    # Deploy 2 wordpress instances and one mysql instance
+    juju deploy wordpress -n 2
+    wait_for "wordpress" "$(idle_condition "wordpress" 0 0)"
+    wait_for "wordpress" "$(idle_condition "wordpress" 0 1)"
+    juju deploy mysql
+    wait_for "mysql" "$(idle_condition "mysql")"
+
+    # Establish relation
+    juju relate wordpress mysql
+
+    # Block until the relation is joined; otherwise, the relation-set commands
+    # below will fail
+    while true; do
+       got=$(juju run --unit 'wordpress/0' 'relation-get --app -r db:2 origin wordpress' || echo 'NOT FOUND')
+       if [ "$got" != "NOT FOUND" ]; then
```

Comment:
nit: we should `${got}`

```suggestion
       if [ "${got}" != "NOT FOUND" ]; then
```